### PR TITLE
chore(dtfs2-7310): fix docs to refer to primitive types and not object instances

### PR DESCRIPTION
**Pull request by AlexBramhill**
## Introduction :pencil2:
We are using the incorrect references in JSdocs for object, boolean, number and string. 
## Resolution :heavy_check_mark:
Update these references

### CODE REVIEW

**Review by JoshBinns2000 — DISMISSED** [review]
LGTM

File: azure-functions/acbs-function/mappings/facility/helpers/get-cover-start-date.js [hunk 1]
```diff
@@ -2,19 +2,19 @@ const { formatDate } = require('../../../helpers/date');
...
  */
 const format = (epoch, fromEpoch) => (fromEpoch ? formatDate(epoch) : epoch);
 
 /**
  * Evaluates the cover start date of a facility.
  * When the product is BSS/EWCS then requestedCoverStartDate
  * property is referred to else coverStartDate.
- * @param {Object} facility Deal's facility object
+ * @param {object} facility Deal's facility object
  * @param {Bool} formatted Return in YYYY-MM-DD date format
```
Comment (abhi-markan) [inline azure-functions/acbs-function/mappings/facility/helpers/get-cover-start-date.js:16]: Can all the `Bool` be updated to `boolean` please across all the files.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
One minor comment to address please.

File: azure-functions/acbs-function/mappings/facility/helpers/get-cover-start-date.js (hunk 1 above)
Comment (AlexBramhill) [reply azure-functions/acbs-function/mappings/facility/helpers/get-cover-start-date.js:16]: updated, and caught a few more that slipped through

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**